Sources/Testing/Issues/Confirmation.swift

```diff
@@ -14,8 +14,7 @@ public struct Confirmation: Sendable {
   ///
   /// This property is fileprivate because it may be mutated asynchronously and
   /// callers may be tempted to use it in ways that result in data races.
-  @Locked
-  fileprivate var count = 0
+  fileprivate var count = Locked(rawValue: 0)
```

Comment:
Can this, and all other remaining `var` variables containing `Locked` types, be `let` now? (I see at least one other, above.)

Reply:
Yes, they should all be `let` now. Good eye.